neuromation/cli/secrets.py

```diff
@@ -0,0 +1,47 @@
+from .root import Root
+from .utils import argument, command, group, pager_maybe
+
+
+@group()
+def secret() -> None:
+    """
+    Operations with secrets.
+    """
+
+
+@command()
+async def ls(root: Root) -> None:
+    """
+    List secrets.
+    """
+
+    ret = []
+    async for secret in root.client.secrets.list():
+        ret.append(secret.key)
+        pager_maybe(ret, root.tty, root.terminal_size)
+
+
+@command()
+@argument("key")
```

Comment:
let's call it `secret_name` not `key`

Reply:
And also: how about to have API like in curl: 
```
$ neuro secret add secret_name --data pas$w0rd
$ neuro secret add secret_name --data @path/to/file.txt
```

Reply:
or the old idea also looks good:
```
$ neuro secret add secret_name pas$w0rd
$ neuro secret add secret_name --from-file path/to/file.txt
```

Reply:
I agree that we need a way to add an arbitrary binary value. Docker always read from a file, so for adding a literal value you need to pass it through stdin:
```
$ printf <secret> | docker secret create my_secret -
```
But I think it would be convenient to support passing it as an argument too. Maybe something like:
```
$ neuro secret add secret_name --from-literal pas$w0rd
$ neuro secret add secret_name --from-file path/to/file.txt
```
We can even add an interactive option (in separate issue):
```
$ neuro secret add secret_name
Enter value of secret_name: ***
Repeat value of secret_name: ***
```